components/gitpod-protocol/src/protocol.ts

```diff
@@ -150,9 +150,10 @@ export interface AdditionalUserData {
     oauthClientsApproved?: { [key: string]: string };
     // to remember GH Orgs the user installed/updated the GH App for
     knownGitHubOrgs?: string[];
-
     // Git clone URL pointing to the user's dotfile repo
     dotfileRepo?: string;
+    // Identifies an explicit team or user ID to which all the user's workspace usage should be attributed to (e.g. for billing purposes)
+    usageAttributionId?: string;
```

Comment:
I think this deserves a separate field on `User` (I'm aware of the overhead). This way we could easier consume it from the `usage` service.

Reply:
Is this a must-have before merging this PR? (Note: This would also require new specific method(s) on the server API.)